managed/services/supervisord/supervisord.go

```diff
@@ -373,6 +375,9 @@ func (s *Service) UpdateLog(offset uint32) ([]string, uint32, error) {
 		line, err := reader.ReadString('\n')
 		if err == nil {
 			newOffset += uint32(len(line))
+			if newOffset-offset > s.gRPCMessageMaxSize {
```

Comment:
What gonna happen in we have line that is longer than `gRPCMessageMaxSize` ?

Reply:
@BupycHuk could you please check this comment?